lang/cs/Org.Apache.REEF.Network.Tests/GroupCommunication/GroupCommunicationTests.cs

```diff
@@ -61,44 +55,6 @@ namespace Org.Apache.REEF.Network.Tests.GroupCommunication
     public class GroupCommunicationTests
     {
         [TestMethod]
-        public void TestSender()
```

Comment:
I think I removed it to test some other things and then forgot to add...lemme add it again